### DISCUSSION TITLE
GFPGAN not working

### DISCUSSION THREAD

**Discussion by Echolink50** [hlky/stable-diffusion-webui#459] [General]
Not sure what I'm doing wrong. Real ESRGAN is working fine but GFPGAN is not loading. I have the model in the right folder. Do I need to rename it or something? Thanks

**Comment by edgartaor** [comment]
I have this files in the folder `./src/gfpgan/experiments/pretrained_models` and it's working
```
  FFHQ_eye_mouth_landmarks_512.pth
  GFPGANv1.3.pth  
  README.md  
  StyleGAN2_512_Cmul1_FFHQ_B12G4_scratch_800k.pth  arcface_resnet18.pth
```
The README.md file have the right links and the GFPGAN model can be downloaded here https://github.com/hlky/stable-diffusion-webui#gfpgan
